src/granite/query/builder.cr

```diff
@@ -99,15 +87,32 @@ class Granite::Query::Builder(Model)
     self
   end
 
+  def and(**matches)
+    and(matches)
+  end
+
+  def and(matches)
+    value = matches[:value]
+    if value.is_a?(Array)
+      and(matches[:field].to_s, matches[:operator], value.compact)
```

Comment:
Shouldn't the default operator be `:in` when it's type is an Array?
```
and(matches[:field].to_s, matches[:operator] || :in, value.compact)
```

Reply:
i added spec.